configure.ac

```diff
@@ -0,0 +1,107 @@
+AC_PREREQ(2.61)
+AC_INIT([pjdfstest],[0.1],)
+AC_CONFIG_AUX_DIR([.])
+AM_INIT_AUTOMAKE
+AC_CONFIG_HEADERS([config.h])
+AC_CONFIG_FILES([ \
+    Makefile \
+])
+
+AC_CANONICAL_HOST
+
+AC_PROG_CC
```

Comment:
There's something really odd/broken with your compiler :(... I don't know what to say.